packages/connector-chrome/tests/events.ts

```diff
@@ -295,7 +295,7 @@ test(`[${name}] Events`, async (t) => {
     }
 
     // List of events that only have to be called once per execution
-    const singles = ['fetch::error', 'scan::start', 'scan::end', 'fetch::end::html'];
+    const singles = ['scan::start', 'scan::end', 'fetch::end::html'];
```

Comment:
question: Why did you remove the `fetch::error`?

Reply:
Because of https://github.com/webhintio/hint/issues/1398#issuecomment-429929512
`fetch::error` is no longer a single event, we receive it twice probably because of speculative downloading.